lib/spack/docs/known_issues.rst

```diff
@@ -0,0 +1,16 @@
+============
+Known issues
+============
+
+Below is the list of known issues:
+
+#. Current conretization algorithm does not take into account default values
```

Comment:
I wouldn't number these issues, as there is no inherent ordering. I think it would be better to format them as sections, like:
```rst
============
Known Issues
============

This is a list of known bugs in Spack. It provides ways of getting around these problems if you encounter them.

-----------------------------------------------------------------
Default variants are not taken into account during concretization
-----------------------------------------------------------------

**Status:** Expected to be fixed in the next release

Insert description of the problem,
example error messages that you might see (so it is easy to search for),
and ways to get around the problem.
```
In each section, we should give a status like "Expected to be fixed in the next release", or "Up for grabs if you want to try to fix it".